src/jit/importer.cpp

```diff
@@ -7081,15 +7080,15 @@ GenTreePtr                Compiler::impFixupStructReturn(GenTreePtr     call,
     call->gtCall.gtReturnType = call->gtType;
 
     // Get the classification for the struct.
-    SYSTEMV_AMD64_CORINFO_STRUCT_REG_PASSING_DESCRIPTOR structDesc;
-    eeGetSystemVAmd64PassStructInRegisterDescriptor(retClsHnd, &structDesc);
-    if (structDesc.passedInRegisters)
+    GenTreeCall* callPtr = call->AsCall();
+    eeGetSystemVAmd64PassStructInRegisterDescriptor(retClsHnd, &(callPtr->structDesc));
+    if (callPtr->structDesc.passedInRegisters)
     {
-        call->gtCall.SetRegisterReturningStructState(structDesc);
+        call->gtCall.SetRegisterReturningStructState(callPtr->structDesc);
 
-        if (structDesc.eightByteCount <= 1)
+        if (callPtr->structDesc.eightByteCount <= 1)
         {
-            call->gtCall.gtReturnType = getEightByteType(structDesc, 0);
+            call->gtCall.gtReturnType = getEightByteType(callPtr->structDesc, 0);
```

Comment:
we don't need to access gtCall here.

callPtr->.gtReturnType = getEightByteType(callPtr->structDesc, 0);